sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/APISpec.groovy

```diff
@@ -88,7 +88,7 @@ class APISpec extends Specification {
 
     static final String garbageLeaseID = UUID.randomUUID().toString()
 
-    public static final String defaultEndpointTemplate = "https://%s.dfs.core.windows.net/"
+    public static final String defaultEndpointTemplate = "http://%s.dfs.core.windows.net/"
```

Comment:
I'm not sure how the Datalake Storage resource was setup but I know the other testing resources are requiring `HTTPS`, if the datalake resources are the same this will cause issues when running tests against the service.

Reply:
Just an accident. I reverted.